roles/python/tasks/main.yml

```diff
@@ -2,16 +2,14 @@
 
 - name: Install some apt packages
   apt:
-    name: '{{ packages }}'
-  vars:
-    packages:
-    - python-dev
-    - python-pip
+    name:
+      - python3-dev
+      - python3-pip
 
 - name: Update pip and setuptools
   pip:
     name: '{{ item }}'
-    state: forcereinstall
+    state: latest
   with_items:
     - pip
```

Comment:
Hopefully, next version of pip module will support multiple entries in `name` :)
Probably coming in Ansible 2.7. My intern/trainee is currently working on this.

Reply:
I will leave it like this for now. thanks :)